packages/flutter_tools/test/integration.shard/test_data/gen_l10n_project.dart

```diff
@@ -608,6 +614,29 @@ void main() {
     "placeholders": {
       "count": {}
     }
+  },
+
+  "vehicleSelect": "{vehicleType, select, sedan{Sedan} cabriolet{Solid roof cabriolet} truck{16 wheel truck} other{Other}}",
```

Comment:
Do you think we need `singleQuote` & `doubleQuote` specific tests in here as well? They exist just a few lines above for plural.

Reply:
Indeed, that discovered a bug, fixed.